content/The_Cartridge_Header.md

```diff
@@ -208,6 +214,11 @@ Specifies the size of the external RAM in the cartridge (if any).
 When using a MBC2 chip $00 must be specified in this entry, even though
 the MBC2 includes a built-in RAM of 512 x 4 bits.
 
+Note: The 2 KB RAM size was not used in any production cartridges.
```

Comment:
Nintendo's docs actually mention that $01 shouldn't be used (see page 299 of [this archived manual](https://ia803208.us.archive.org/9/items/GameBoyProgManVer1.1/GameBoyProgManVer1.1.pdf)), so I'm not entirely sure where the "2 KB" number came from.

Reply:
That's a good point - it does follow the x4 progression of the other values, but maybe it should just be noted as invalid?

Reply:
As noted in https://github.com/gbdev/pandocs/pull/108#discussion_r567311144, it's probably an early mistake. That said, since this line is being taken care of by the other PR, I would rather drop it from this one, to avoid conflicts.